src/client/components/additionalConfigs/volumeSlider/volumeSlider.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import Slider from 'react-rangeslider';
+
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faVolumeUp } from '@fortawesome/free-solid-svg-icons';
+import 'react-rangeslider/lib/index.css';
+import './volumeSlider.css';
+
+const volumeIcon = <FontAwesomeIcon icon={faVolumeUp} />;
+
+class VolumeSlider extends React.Component {
+  constructor (props, context) {
+    super(props, context);
+    this.state = {
+      value: 20,
+      maxValue: 100,
+      minValue: 0
+    };
+  }
+  handleOnChange = (value) => {
```

Comment:
Better to name your function parameters, constants, etc more informatively, for eg., sliderValue